app/frontend/actions/accounts.ts

```diff
@@ -54,7 +60,7 @@ export default (store: Store) => {
       targetAccountIndex: accountIndex,
     })
     const targetAddress = await getWallet()
-      .getAccount(accountIndex)
+      ?.getAccount(accountIndex)
```

Comment:
the wallet object shouldn't be undefined/null here, otherwise it could lead to inonsisten behavior down the line. As suggested in another comment, I'd rather introduce a getWalletOrThrow() method which would assert that the wallet is properly loaded